plottable.d.ts

```diff
@@ -3578,7 +3578,7 @@ declare module Plottable {
 
 declare module Plottable {
     module Dispatchers {
-        type KeyCallback = (keyCode: number, event: KeyboardEvent) => void;
```

Comment:
it's optional cuz we don't use event in keyInteraction (sorry forgot to revert that in this commit, it might make more sense after the revert)